jobs/JEVS_CAM_STATS

```diff
@@ -138,7 +139,7 @@ fi
 ####################################
 if [ $VERIF_CASE = radar ] || [ $VERIF_CASE = severe ]; then
    $HOMEevs/scripts/${STEP}/${COMPONENT}/exevs_${COMPONENT}_${VERIF_CASE}_${STEP}.sh
-elif [ $MODELNAME = nam_firewxnest ]; then
+elif [ $MODELNAME = nam_firewxnest ] || [ $MODELNAME = rrfs_firewxnest ]; then
```

Comment:
The driver scripts and ecf scripts have the MODELNAME changed to firewxnest. Where do the nam_firewxnest and rrfs_firewxnest overwrite this in the jobcard, for example in JEVS_CAM_STATS?

Reply:
Yes, MODELNAME is changed to "firewxnest", but only for the firewxnest plots job.  For the two firewxnest stats jobs (nam_firewxnest and rrfs_firewxnest),  the MODELNAME is set to "nam_firewxnest" and "rrfs_firewxnest" respectively.  That's why in this JEVS_CAM_STATS J-job script, the MODELNAME is able to match with either name when firewx stats jobs are run.  

In this PR, the nam_firewxnest plots job was expanded to include rrfs_firewxnest, so the MODELNAME was expanded to just "firewxnest".  The nam_firewxnest _stats_ job was not expanded, but an additional "rrfs_firewxnest" stats job was added.